.gitignore

```diff
@@ -521,3 +521,7 @@ api/api-packages-v2/out/
 api/api-packages-v2/target/
 api/api-packages-v2/logs/
 api/api-packages-v2/config/
+api/api-internal-pipeline-structure-v2/out/
```

Comment:
as we're introducing v2 of internal pipeline-structure API, should we remove `v1`?

Reply:
Decided against bumping up the api. Updated the v1 itself to return the desired values.